### DISCUSSION TITLE
How to setup local database

### DISCUSSION THREAD

**Discussion by aasenomad** [realworld-apps/realworld#771] [Q&A]
I look over Read Me but don't see any instruction on how to set up local database using MySQL. Also, I'm not sure what tables, columns, row, and value I need to create so it'll great if there is any instructions for a newbie like me.

**Comment by jt6677** [comment]
look at others' example, some of them have sql in them.  https://github.com/fdaciuk/conduit-api     it has prisma. so it has migration.sql

**Comment by geromegrignon** [comment]
Thanks for your answer @jt6677.

For the moment I advise you to do so.
At some point in the future, I **really** want to add instructions in the documentation not to act as a gatekeeper for new developers.

**Reply by jt6677** [reply]
some examples use some kind of ORM which would create tables automatically. some creates table manually, some uses some kind of migration tools. I like the variety but this can be difficult for ppl who started programming.

**Comment by cirosantilli** [comment]
Any sane example will at most require you to create databases manually, and then the code will create its own tables, possibly upon running a migration script, but that is very implementation dependent, and can only be explained on the implementation specific docs.

What we need is to point out a reference front and backend implementations (maintainers are currently working on that now I believe), and then say how to launch both locally on the main project README.

The best local setup is sqlite which does not require manual database creation at all as all goes in a single file (and the reference implementation is going to support that).

For now you just have to search how to setup a database of a given type on stack overflow, and from documentation/source of implementations understand what has to be the database name or how to set the name in the project configs.
